m3-sys/cminstall/src/config/I386_DJGPP

```diff
@@ -83,6 +84,9 @@ proc compile_c(source, object, options, optimize, debug) is
 end
 
 proc m3_link(prog, options, objects, imported_libs, shared) is
+  imported_libs = ConvertLibsToStandalone(imported_libs, shared)
+  imported_libs = escape(subst_chars(imported_libs, "\\", "/"))
+  objects       = escape(subst_chars(objects, "\\", "/"))
```

Comment:
Why not just:
```
imported_libs = subst_chars(imported_libs, "\\", "/")
objects = subst_chars(objects, "\\", "/")
```
?

Reply:
I.e. without
imported_libs = ConvertLibsToStandalone(imported_libs, shared)
?
 
I simply was not tested this variant. I think it can be worked / good variant too

I can prepare another pool request